File: app/src/models/vote.ts
export enum VoteValue {
    VoteValueYes = 1,
    VoteValueMaybe = 2,
    VoteValueNo = 3,
}

export interface Vote {
    optionKey: string;
    voterName: string;
    value: VoteValue;
    created: string;
}

File: src/models/vote.go
package models

import "time"

const (
	VoteValueYes   = 1
	VoteValueMaybe = 2
	VoteValueNo    = 3
)

type Vote struct {
	GroupKey  string `json:"-" db:"group_key"`
	OptionKey string `json:"optionKey" db:"option_key"`

	VoterName string `json:"voterName" db:"voter_name"`

	Value int `json:"value" db:"value"`

	Created time.Time `json:"created" db:"created"`
}
